e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -0,0 +1,711 @@
+      );
+      cy.location().should(({ hash, pathname }) => {
+        expect(pathname).to.equal("/question");
+
+        const card = deserializeCardFromUrl(hash);
+        expect(card.name).to.deep.equal(TARGET_QUESTION.name);
+        expect(card.display).to.deep.equal(TARGET_QUESTION.display);
+        expect(card.dataset_query.query).to.deep.equal({
+          ...TARGET_QUESTION.query,
+          filter: QUERY_FILTER_CREATED_AT,
+        });
+      });
+    });
+
+    it("allows setting saved question with multiple parameters as custom destination", () => {
+      cy.createQuestion(TARGET_QUESTION);
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      addSavedQuestionDestination();
+      addSavedQuestionCreatedAtParameter();
+      addSavedQuestionQuantityParameter();
+      cy.get("aside").button("Done").click();
+
+      saveDashboard();
+
+      clickLineChartPoint();
+      cy.findByTestId("qb-filters-panel").should(
+        "contain.text",
+        "Created At is August 1–31, 2022",
+      );
+      cy.findByTestId("qb-filters-panel").should(
+        "contain.text",
+        "Quantity is equal to 79",
+      );
+      cy.location().should(({ hash, pathname }) => {
+        expect(pathname).to.equal("/question");
+        const card = deserializeCardFromUrl(hash);
+        expect(card.name).to.deep.equal(TARGET_QUESTION.name);
+        expect(card.display).to.deep.equal(TARGET_QUESTION.display);
+        expect(card.dataset_query.query).to.deep.equal({
+          ...TARGET_QUESTION.query,
+          filter: ["and", QUERY_FILTER_CREATED_AT, QUERY_FILTER_QUANTITY],
+        });
+      });
+    });
+
+    it("does not allow setting saved question as custom destination if user has no permissions to it", () => {
+      cy.createCollection({ name: RESTRICTED_COLLECTION_NAME }).then(
+        ({ body: restrictedCollection }) => {
+          cy.updateCollectionGraph({
+            [USER_GROUPS.COLLECTION_GROUP]: {
+              [restrictedCollection.id]: "none",
+            },
+          });
+
+          cy.createQuestion({
+            ...TARGET_QUESTION,
+            collection_id: restrictedCollection.id,
+          });
+        },
+      );
+
+      cy.signOut();
+      cy.signInAsNormalUser();
+
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: card }) => {
+          visitDashboard(card.dashboard_id);
+        },
+      );
+
+      editDashboard();
+
+      getDashboardCard().realHover().icon("click").click();
+      cy.get("aside").findByText("Go to a custom destination").click();
+      cy.get("aside").findByText("Saved question").click();
+
+      modal().findByText(RESTRICTED_COLLECTION_NAME).should("not.exist");
+      modal().findByText(TARGET_QUESTION.name).should("not.exist");
```

Comment:
This is similar to another comment I made for setting dashboards as a custom destination. I think it's a bit misleading since the target question will never be shown here even if users have access to the collection since they need to navigate into the collection first to see the target question there.

I think we should remove this assertion.

Reply:
e24965ec2ab309ef54a09066077920443cb2c3cb